src/lib/dataviews/mode/DataViewLocal.ts

```diff
@@ -53,15 +53,19 @@ export class DataViewLocal extends DataViewMode {
   }
 
   private async groupBy(operationColumn: string, operation: AggregationType) {
-    const sourceData = await this.getSourceData([operationColumn]);
-    const { groups, nullCount } = groupValuesByAnotherColumn(
-      sourceData,
-      operationColumn,
-      this.column
-    );
+    let groupedValues;
+
+    if (operationColumn) {
+      const sourceData = await this.getSourceData([operationColumn]);
+      groupedValues = groupValuesByAnotherColumn(sourceData, operationColumn, this.column);
+    } else {
+      const sourceData = await this.getSourceData();
+      groupedValues = groupValues(sourceData, this.column);
```

Comment:
Not sure about this approach. It's true that the grouping is kinda different because this one returns a number, but I would simplify that until we see a performance or memory problem.

It would use the same function to group them both, whether operation is count or not.

So, I would perform these changes:
- Remove `groupValues` function.
- Renaming `groupValuesByAnotherColumn` to `groupValuesByColumn`
- Refactor this code so that could be used in both cases like:

```js
const sourceData = await this.getSourceData([ operationColumn || this.column ]);
const { groups, nullCount } = groupValuesByColumn(
  sourceData,
  operationColumn || this.column,
  this.column
);
```